amoro-core/src/main/gen-java/org/apache/amoro/api/CommitMetaProducer.java

```diff
@@ -11,7 +11,8 @@
 public enum CommitMetaProducer implements org.apache.amoro.shade.thrift.org.apache.thrift.TEnum {
   OPTIMIZE(0),
   INGESTION(1),
-  DATA_EXPIRATION(2);
+  DATA_EXPIRATION(2),
+  CLEAN_DANGLING_DELETE(3);
```

Comment:
We may need to change the thrift code and rebuild from it:
https://github.com/apache/amoro/blob/master/amoro-core/src/main/thrift/amoro_table_metastore.thrift#L99